src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -261,6 +272,20 @@ public VillagerProfession(String name, String texture)
             this.texture = new ResourceLocation(texture);
             ((RegistryDelegate.Delegate<VillagerProfession>)delegate).setName(name);
         }
+        
+        // ===== Changes by dvatreknerd314 =====
+        
+        // The idea behind this is to be able to pass in a triple (or array of triples) to VillagerRegistry
+        // to set the trades for combinations of profession and career.  Could potentially be used to
+        // override vanilla trades
+        //
+        // This returns a static copy of the careers list to get the number of careers and get a career's
+        // trading array
+        public static List<VillagerCareer> getCareers()
+        {
+            return careers;
```

Comment:
Giving raw access to internal arrays is a bad thing, modders will fuck it up and break the registry. Making it public promotes and condones this, don't do it.